Add Project Fugu report

tomayac opened the pull request:
This PR adds a new top-level report for Project Fugu APIs. It is sorted by the level of maturity of the APIs, starting from (🐡 Launched), over to (🧪 Origin trial), and ending with (🚩 Behind flag).

Some of the reports show no data, since the number of pages that implement these APIs currently is still low. I round to 5 digits in the queries, which, if my math is right and assuming the current size of 5,350,650 URLs of the mobile archive, that there must be at least 0.00001 * 5,350,650 ≈ 54 pages using an API in order for the report to show data.

Please review this jointly with https://github.com/HTTPArchive/httparchive.org/pull/206.

rviscomi commented on sql/timeseries/asyncClipboardRead.sql:
```diff
@@ -0,0 +1,18 @@
+  IF(ENDS_WITH(_TABLE_SUFFIX, 'desktop'), 'desktop', 'mobile') AS client,
+  ROUND(SUM(IF(IFNULL(
+    JSON_EXTRACT(payload, '$._blinkFeatureFirstUsed.Features.AsyncClipboardAPIRead'),
+    JSON_EXTRACT(payload, '$._blinkFeatureFirstUsed.Features.2369'))
```
We could query only the blink features by using the `httparchive.blink_features` table directly. For example:

```sql
SELECT * FROM `httparchive.blink_features.usage` WHERE id = '2369'
```

This table is very small, so much more efficient. Could you update the queries like this one to use the `blink_features` table?

rviscomi replied:
Also consider including `num_urls` in the results. Even though the `percent` value is graphed, many metrics' percents are 0.0% so it may be helpful to drill down into absolute counts in the data table:

![image](https://user-images.githubusercontent.com/1120896/88328790-f30da400-ccf6-11ea-9f46-a98f184ab323.png)

tomayac commented:
@rviscomi Thanks for the review. I have moved every query over to use the `httparchive.blink_features.usage` table, and now I also added `num_urls` and `sample_urls`, so we can debug further.

tomayac commented:
FYI: @christianliebel Once this lands, the features for the Almanac chapter are stable. We may want to consider commenting out features that require a user gesture, but for now, the report is stable. Given that we now use a new table, the user gesture requirement may go away in the future, depending on how the table is populated.

rviscomi commented on sql/timeseries/asyncClipboardRead.sql:
```diff
@@ -0,0 +1,23 @@
+  client,
+  num_urls,
+  ROUND(num_urls / total_urls * 100, 5) AS percent,
+  ANY_VALUE(sample_urls) sample_urls,
```
If `sample_urls` would be useful, could we link the metric description to the corresponding Chrome Status dashboard? Otherwise I don't think the reporting UI can support an array of URLs.

tomayac replied:
We certainly could. The URL structure is https://chromestatus.com/metrics/feature/timeline/popularity/${featureID}. Should I remove `sample_urls` from the queries again?

rviscomi replied:
Yes please 😅 

`num_urls` is a simple value we can show in the table so there's no problem keeping that.

tomayac replied:
Done ✅

rviscomi replied:
Great thanks!
